src/components/data_search/DatasetFilterList.jsx

```diff
@@ -42,6 +45,38 @@ export const FilterItemList = (props) => {
   );
 };
 
+export const AccessManagementFilterItemList = (props) => {
```

Comment:
Instead of defining a new function, is there a way to combine this with `FilterItemList`? If an additional prop is an optional function that converts `filterName` to an image source, I think both cases could be handled in the same function, and the duplicate code can be avoided.